proto/substrait/algebra.proto

```diff
@@ -88,7 +88,8 @@ message ReadRel {
 
   // A table composed of literals.
```

Comment:
minor/nit: For clarity it would be good to update this to
```
A table composed of expressions.
```

Reply:
fixed it